### PR TITLE
Update eslint to 0.21.0.

### PR DESCRIPTION
**Pull request by anselmbradford**
Updates grunt-eslint to 13.0.0., which includes eslint 0.21.0.

## Changes

- Updates eslint config to add additional rules in ESLint 0.21.0.
- Refactors `/the-bureau/index.js` to remove inline event handler functions and removes redundant DOM lookup.
- Fixes minor ESLint offenses.

## Testing

- Run `grunt lintjs --quiet` and see no errors.

## Review

- @sebworks 
- @KimberlyMunoz 
- @jimmynotjim

### CODE REVIEW

**Comment by sebworks** [comment]
Why create the functions (_enterBreakpoint ) outside the if statement when they are referencing something that is only defined in the if statement (bureauMissionDom) ?

**Comment by anselmbradford** [comment]
> Why create the functions (_enterBreakpoint ) outside the if statement when they are referencing something that is only defined in the if statement (bureauMissionDom) ?

I thought the same thing, but we're enforcing http://eslint.org/docs/rules/no-inner-declarations.html, which is required in parsers earlier than ES6. I don't think there's a difference really if they're inside or outside the if statement as they won't be called until needed, however, I believe declaring them in the root scope leaves the door open that they can be used as event handlers for multiple code blocks, if needed.

**Comment by sebworks** [comment]
Should we be passing the reference instead of using a closure?

**Comment by sebworks** [comment]
:+1:

**Comment by anselmbradford** [comment]
> Should we be passing the reference instead of using a closure?

In https://github.com/cfpb/cfgov-refresh/pull/537/files#diff-03f2e58f44995a5595972ef18e306eb5R20 you mean? It doesn't appear a closure is necessary for those code blocks. Personally I avoid closures as much as I can because they can make scope hard to follow, so for clarity and DRYness (if a function is reused) I tend to call event listeners by reference.

**Comment by anselmbradford** [comment]
Forgot to note, `npm update` is required!
